Data/Snoverism-data.json

```diff
@@ -148,6 +148,10 @@
     {
       "Source": "https://www.youtube.com/watch?v=nJpyUYZrSBY&t=264s",
       "Quote": "The key thing to learning Powershell is to have a sense of adventure and a sense of joy in exploring the system."
+    },
+    {
+      "Source": "https://twitter.com/jsnover/status/1588180376488640512s",
```

Comment:
Would you mind double check the link?
Perhaps you mean this one without the trailing `s`? `https://twitter.com/jsnover/status/1588180376488640512`.

Reply:
Probably fat fingered it, your'e right!